src/vorrichtung/components/grid/views.cljs

```diff
@@ -0,0 +1,108 @@
+(ns vorrichtung.components.grid.views
+  (:require
+    [clojure.string :refer [join]]
+    [re-frame.core :refer [subscribe dispatch]]
+    [vorrichtung.components.grid.db :refer [config->order-class order->order-column-map item-value previous-page?
+                                            next-page?]]
+    [vorrichtung.html :refer [safe-html]]
+    [vorrichtung.utils :refer [css-class]]))
+
+
+(def ordering->class
+  {:ASC "sorted"
+   :DESC "sorted-reverse"
+   :NONE nil})
+
+
+(defn th
+  [config column column-order-index mappings]
+  [:th {:class (:id column)}
+   (if (:orderable_by column)
+     [:div
+      {:on-click #(dispatch [(get-in mappings [:events :order-by-column]) config (:orderable_by column)])}
+      [:span {:class (css-class ["sortable" (config->order-class config (:orderable_by column) ordering->class)])}
+       (:verbose_name column)]
+      [:div.sort-direction]]
+     (:verbose_name column))])
```

Comment:
Tohle mi připadá pěkný, jak se to rozpadlo na jednotlivé komponenty, které se jen poskládají do sebe v jedné funkci.